synapse/push/httppusher.py

```diff
@@ -294,8 +294,13 @@ def dispatch_push(self, event, tweaks, badge):
             defer.returnValue([])
         try:
             resp = yield self.http_client.post_json_get_json(self.url, notification_dict)
+            logger.info(
+                "Sent to: %s push-notify: %r get response: %r" % (
+                    self.url, notification_dict, resp
+                )
+            )
         except:
-            logger.warn("Failed to push %s ", self.url)
+            logger.warn("Failed to push %s %r" % (self.url, notification_dict))
```

Comment:
For me it does not matter the level at which these messages will be registered, but since this is required for debugging your proposal is logical.